vcf-dumper/vcf-dumper-ws/src/main/java/uk/ac/ebi/eva/vcfdump/server/rest/HtsgetVcfController.java

```diff
@@ -53,145 +57,170 @@
 
     private Properties evaProperties;
 
-    @Autowired
     private VariantSourceService variantSourceService;
-    @Autowired
+
     private VariantWithSamplesAndAnnotationsService variantService;
 
-    public HtsgetVcfController() throws IOException {
+    public HtsgetVcfController(VariantSourceService variantSourceService,
+                               VariantWithSamplesAndAnnotationsService variantService) throws IOException {
+        this.variantSourceService = variantSourceService;
+        this.variantService = variantService;
         evaProperties = new Properties();
         evaProperties.load(VcfDumperWSServer.class.getResourceAsStream("/eva.properties"));
     }
 
     @RequestMapping(value = "/{id}", method = RequestMethod.GET, consumes = "application/*",
             produces = "application/vnd.ga4gh.htsget.v0.2rc+json; charset=UTF-8")
     public ResponseEntity getHtsgetUrls(
+            @ApiParam(value = "Identifiers of studies, e.g. PRJEB9799. Each individual identifier of " +
```

Comment:
If several studies can be specified here, could we say "Study identifiers (separate with comma for multiple studies)"? Also "identifier of studies" can be re-phrased to "study identifiers"...